packages/api/src/apps/api/app.ts

```diff
@@ -56,6 +69,8 @@ async function run(env: ProcessEnv) {
       services.registry(libs.lastBlock, blockNumber).catch(console.error);
       services.emps(libs.lastBlock, blockNumber).catch(console.error);
       libs.lastBlockUpdate = blockNumber;
+      // price updates need to be throttled, coingecko will rate limit
+      services.prices.update().catch(console.error);
```

Comment:
do you not need to wait for this promise to resolve before proceeding?

Reply:
in this case its being triggered by a block event, so i just let it run asyncronously, dont want to block the block events.